packages/components/src/summary/index.js

```diff
@@ -3,59 +3,44 @@
  * External dependencies
  */
 import { __ } from '@wordpress/i18n';
-import classnames from 'classnames';
 import { Children, cloneElement } from '@wordpress/element';
-import { Dropdown, NavigableMenu } from '@wordpress/components';
+import { Dropdown } from '@wordpress/components';
 import PropTypes from 'prop-types';
-import { uniqueId } from 'lodash';
 import { withViewportMatch } from '@wordpress/viewport';
 
+/**
+ * Internal dependencies
+ */
+import Menu from './menu';
+
 /**
  * A container element for a list of SummaryNumbers. This component handles detecting & switching to
  * the mobile format on smaller screens.
  *
  * @return { object } -
  */
 const SummaryList = ( { children, isDropdownBreakpoint, label } ) => {
-	if ( ! label ) {
-		label = __( 'Performance Indicators', 'wc-admin' );
-	}
-
+	const items = children( {} );
```

Comment:
Optional nitpick:

Reply:
The child function of `report-summary/index.js` does however. I'll get `Cannot read property 'onToggle' of undefined` if I remove this argument.